saleor/urls.py

```diff
@@ -26,7 +26,7 @@
 non_translatable_urlpatterns = [
     url(r'^dashboard/',
         include((dashboard_urls, 'dashboard'), namespace='dashboard')),
-    url(r'^graphql/', csrf_exempt(GraphQLView.as_view(
+    url(r'^graphql', csrf_exempt(GraphQLView.as_view(
```

Comment:
Why would you like to change that? IMHO there should be exactly one way to access an API and Saleor terminates all other resource names with a slash.

Reply:
It is because ```graphql``` url make graphql can be accessed by both ```graphql``` and ```graphql\```. With ```graphql\```, post to ```graphql``` will be lost even it is redirected by ```APPEND_SLASH``` settings.

Sometime it will confuse user who don't know Saloer's graphql api must be ```graphql\``` instead of ```graphql```